lib/rate-limiter.js

```diff
@@ -154,9 +127,10 @@ exports.epochMs = function(appName, key, client, callback){
  * @param {string} appName The name of the app you want to retrieve the apps from
  * @param {string} key The key the app is looking to rate limit on
  * @param {integer} cost How many tokens this request costs
+ * @param {integer} rateLimit How many requests per time period this key should get
  * @param {function} callback signature (err, response) response is the amount of tokens left. -1 means that the request was disallowed
```

Comment:
Wait no this is separate from what's actually getting sent to the LUA script, got it